lib/tasks/auth_source_external.rake

```diff
@@ -0,0 +1,23 @@
+desc 'Create Auth Source External'
+namespace :auth_source_external do
+  desc 'create auth source external'
+  task :create => :environment do
+    User.as_anonymous_admin do
+      external_name = Setting[:authorize_login_delegation_auth_source_user_autocreate]
```

Comment:
Why are we using this setting here rather than just creating an auth source with a predefined name if none exists?

Reply:
Since our documentation mentions `External` to be set. I will create an auth source withe the `External` name.